### PR TITLE
sync en /docs/concepts/services-networking/endpoint-slices.md and /do…

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/k8smeetup/website-tasks/issues/3681
ref:https://github.com/k8smeetup/website-tasks/issues/3676

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -112,6 +119,14 @@ platform for additional features such as topological routing.
 
 当涉及如何路由内部流量时，Endpoint Slices 可以充当 kube-proxy 的真实来源。启用该功能后，在服务的 endpoints 规模庞大时会有可观的性能提升。
 
+## Address Types
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/endpoint-slices.md]: 地址类型

**Comment by tanjunchen** [comment]
@tengqm 对于有改进的汉化，等下我在新建一个 pr 统一完善下，谢谢

**Comment by tengqm** [comment]
@tanjunchen No worries.

**Pull request merged**